modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/dump/IgniteCacheDumpSelfTest.java

```diff
@@ -146,6 +146,23 @@ public void testCacheDump() throws Exception {
         }
     }
 
+    /** */
+    @Test
+    public void testZippedCacheDump() throws Exception {
+        snpPoolSz = 4;
+
+        try {
+            IgniteEx ign = startGridAndFillCaches();
+
+            createDump(ign, DMP_NAME, null, true);
+
+            checkDump(ign);
```

Comment:
We must ensure here that dump directory conatins zip files only.

Reply:
Added checkPartitionsNames